src/commands/verify.ts

```diff
@@ -0,0 +1,24 @@
+import { CommandInteraction, EmbedBuilder, SlashCommandBuilder } from "discord.js";
+
+export default {
+    data: new SlashCommandBuilder().setName("verify").setDescription("You should verify yourself NOW!"),
+    async execute(interaction: CommandInteraction) {
+        const embeds = new EmbedBuilder()
+            .setTitle("UofG Racing Verification")
+            .setDescription("Welcome! To gain access to the server, please verify yourself.")
+            .addFields(
+                { name: "How", value: "1. Apply to this [form](<https://youtu.be/fC7oUOUEEi4>).\n2. **DM the bot** the email given to the form.\n3. Follow the instructions given." },
```

Comment:
Add another point here that tells them to sign up to gryphlife (I don't have a url for you as we missed registration for the summer (we're working on that)

Reply:
I'll place a temporary header [link](https://www.youtube.com/watch?v=dQw4w9WgXcQ) for now.